src/__tests__/Login.test.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { screen, render, fireEvent, waitFor } from '@testing-library/react';
+import { FusionAuthLoginButton } from '../components/FusionAuthLoginButton';
+import { FusionAuthProvider } from '../providers/FusionAuthProvider';
+import { createHash } from 'crypto';
+
+afterEach(() => {
+    jest.clearAllMocks();
+});
+
+Object.defineProperty(global.self, 'crypto', {
+    value: {
+        subtle: {
+            digest: (algorithm: string, data: Uint8Array) => {
+                return new Promise(resolve =>
+                    resolve(
+                        createHash(algorithm.toLowerCase().replace('-', ''))
+                            .update(data)
+                            .digest(),
+                    ),
+                );
+            },
+        },
+        getRandomValues: array => {
+            for (const num of array) {
+                return Math.random() * num;
+            }
+        },
+    },
+});
```

Comment:
Let's move this to a function in a new directory `__tests__/mocks` called `mockCrypto` and then we can just call that function in our test or `beforeEach` block